pallets/assets-info/src/lib.rs

```diff
@@ -71,7 +72,7 @@ decl_storage! {
                         name: name.clone(),
                         symbol: token.clone(),
                         description: description.clone(),
-                        decimals: decimals.clone(),
+                        decimals: decimals.to_owned(),
```

Comment:
Per my understanding, `.clone()` or `.to_owned()` is required because the value was borrowed. Why in case of `name`, `symbol` and `description` the right way to go is `.clone()` and for `decimals` it is `.to_owned()` ?